src/blockdata/transaction.rs

```diff
@@ -241,7 +241,7 @@ impl Default for TxOut {
 /// for 0-input transactions, which results in unambiguously parseable transactions.
 #[derive(Clone, PartialEq, Eq, Debug, Hash)]
 pub struct Transaction {
-    /// The protocol version, should always be 1.
+    /// The protocol version, could be 1 or 2 (BIP 68).
```

Comment:
Minor nit: shouldn't it say `is currently expected to be 1 or 2 (BIP 68).` since bigger versions are possible and valid too?

Reply:
Right, updated, even if bigger versions aren't standard